meson.build

```diff
@@ -0,0 +1,151 @@
+project('pyfmmlib', 'c', 'fortran',
+  version : '2023.1',
+  license: 'MIT',
+  meson_version: '>=0.64.0',
+  default_options : ['warning_level=2', 'fortran_std=legacy'],
+)
+
+# {{{ gather includes
+
+py_mod = import('python')
+py = py_mod.find_installation(pure: false)
+py_dep = py.dependency()
+
+# NOTE: meson does not like absolute paths, so we dance around it
```

Comment:
Not even for include files? That seems strange.

Reply:
Mostly stole this from `scipy`
https://github.com/scipy/scipy/blob/5e3f6da3c9b3c7707a38ef99a5aa8d7f53e824a7/scipy/meson.build#L48

My understanding is that global packages are found with `pkg-config` or with a special `dependency` command in meson, so just listing global paths doesn't work.